test/dotnet/input/InsertBulk/insertBulkErrors.txt

```diff
@@ -555,4 +555,77 @@ drop table sourceTable
 drop table sourceTable1
 drop table destinationTable
 
- SET implicit_transactions OFF
\ No newline at end of file
+ SET implicit_transactions OFF
+
+ # identity
```

Comment:
Add a comment about identity errors testing at the top of this file as well

Reply:
Done